src/components/toolbar/toolbar.spec.js

```diff
@@ -291,10 +283,19 @@ describe('<md-toolbar>', function() {
       }
 
       element = $compile(template)(pageScope);
-      controller = element.controller('mdToolbar');
+      controller = element.find('md-toolbar').controller('mdToolbar');
 
       pageScope.$apply();
       $timeout.flush();
     });
   }
+
+  function getTransform (st) {
+    return st.getPropertyValue("-webkit-transform") ||
+           st.getPropertyValue("-moz-transform") ||
+           st.getPropertyValue("-ms-transform") ||
+           st.getPropertyValue("-o-transform") ||
+           st.getPropertyValue("transform") ||
+           'none';
```

Comment:
This function can be replaced with an util constant.

https://github.com/angular/material/blob/master/src/core/util/constant.js#L39
`$mdConstant.CSS.TRANSFORM`